<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetTesterClass</class>
 <widget class="QMainWindow" name="NetTesterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBoxOperation">
      <property name="title">
       <string>操作区</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxReceive">
         <property name="title">
          <string>接收网络测试数据</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editPort"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="buttonListenTCP">
            <property name="text">
             <string>监听网络数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSend">
         <property name="title">
          <string>发送网络测试数据</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editSendIP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editSendPort"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="buttonConnect">
            <property name="text">
             <string>建立链接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSendTCP">
            <property name="text">
             <string>发送数据包(第一次发会建立链接)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonTest">
            <property name="text">
             <string>循环发送txt测试数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxData">
      <property name="title">
       <string>数据交换区</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditData"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxLog">
      <property name="title">
       <string>日志区</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditLog"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NetTester.qrc"/>
 </resources>
 <connections/>
</ui>
